app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app import cafe, errors
+
+
+def go_to_cafe(friends: list, cafe: cafe.Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except errors.VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
When a `VaccineError` is caught, the function immediately returns "All friends should be vaccinated". This means that if any friend is not vaccinated, the function will not check the rest of the friends. If the task requires checking all friends before returning a message, consider collecting errors and returning a summary after the loop.